src/main/java/frc/robot/Arm.java

```diff
@@ -17,21 +18,26 @@ public class Arm extends SubsystemBase {
     private CANSparkMax motorTwo;
     private PIDController pid;
     private DigitalInput homeSwitch;
+    private boolean broken;
+    private EncoderSim simEncoder;
 
     public Arm() {
         homeSwitch = new DigitalInput(Constants.Arm.DIGITAL_INPUT_CHANNEL);
         pid = new PIDController(Constants.Arm.P_COEFFICIENT, Constants.Arm.I_COEFFICIENT, Constants.Arm.D_COEFFICIENT);
         armEncoder = new Encoder(Constants.Arm.ENCODER_CHANNEL_A, Constants.Arm.ENCODER_CHANNEL_B);
         motorOne = new CANSparkMax(Constants.Arm.ARM_MOTOR_ONE, MotorType.kBrushless);
         motorTwo = new CANSparkMax(Constants.Arm.ARM_MOTOR_TWO, MotorType.kBrushless);
+        broken = false;
         if (Robot.isSimulation()) {
             REVPhysicsSim.getInstance().addSparkMax(motorOne, DCMotor.getNEO(1));
             REVPhysicsSim.getInstance().addSparkMax(motorTwo, DCMotor.getNEO(1));
+            simEncoder = new EncoderSim(armEncoder);
         }
+        
     }
 
     public double getAngle() {
-        return ((double) armEncoder.get() / Constants.Arm.ENCODER_COUNT) * 360;
+        return ((double) simEncoder.getCount() / Constants.Arm.ENCODER_COUNT) * 360;
```

Comment:
Unless I am mistaken I think you should only return the simEncoder value if you are in simulation mode, otherwise return the armEncoder (real encoder) value. Other than this everything looks great.